examples/app.js

```diff
@@ -459,6 +485,151 @@ fields.forEach(({ field, type, name }) => {
+            return formValues.acceptRelayCandidates;
+        case 'prflx':
+            return formValues.acceptPrflxCandidates;
+        default:
+            console.warn('ShouldAcceptICECandidate: Unknown candidate type:', candidate.type);
+            return false;
+    }
+}
+
+$('#natTraversalEnabled').on('click', () => {
+    $('#accept-host').prop('checked', true);
+    $('#send-host').prop('checked', true);
+    $('#accept-relay').prop('checked', true);
+    $('#send-relay').prop('checked', true);
+    $('#accept-srflx').prop('checked', true);
+    $('#send-srflx').prop('checked', true);
+    $('#accept-prflx').prop('checked', true);
+    $('#send-prflx').prop('checked', true);
+
+    saveAdvanced();
+});
+
+$('#forceSTUN').on('click', () => {
+    $('#accept-host').prop('checked', false);
+    $('#send-host').prop('checked', false);
+    $('#accept-relay').prop('checked', false);
+    $('#send-relay').prop('checked', false);
+    $('#accept-srflx').prop('checked', true);
+    $('#send-srflx').prop('checked', true);
+    $('#accept-prflx').prop('checked', false);
+    $('#send-prflx').prop('checked', false);
+
+    saveAdvanced();
+});
+
+$('#forceTURN').on('click', () => {
+    $('#accept-host').prop('checked', false);
+    $('#send-host').prop('checked', false);
+    $('#accept-relay').prop('checked', true);
+    $('#send-relay').prop('checked', true);
+    $('#accept-srflx').prop('checked', false);
+    $('#send-srflx').prop('checked', false);
+    $('#accept-prflx').prop('checked', false);
+    $('#send-prflx').prop('checked', false);
+
+    saveAdvanced();
+});
+
+$('#natTraversalDisabled').on('click', () => {
+    $('#accept-host').prop('checked', true);
+    $('#send-host').prop('checked', true);
+    $('#accept-relay').prop('checked', true);
+    $('#send-relay').prop('checked', true);
+    $('#accept-srflx').prop('checked', true);
+    $('#send-srflx').prop('checked', true);
+    $('#accept-prflx').prop('checked', true);
+    $('#send-prflx').prop('checked', true);
+
+    saveAdvanced();
+});
+
+function saveAdvanced() {
+    $('#accept-host').trigger('change');
+    $('#send-host').trigger('change');
+    $('#accept-relay').trigger('change');
+    $('#send-relay').trigger('change');
+    $('#accept-srflx').trigger('change');
+    $('#send-srflx').trigger('change');
+    $('#accept-prflx').trigger('change');
+    $('#send-prflx').trigger('change');
+}
+
+/**
+ * Determines whether the ICE Candidate should be sent to the peer.
+ * @param formValues Settings used.
+ * @param candidate {RTCIceCandidate} iceCandidate to check
+ * @returns true if the candidate should be sent to the peer.
+ */
+function shouldSendIceCandidate(formValues, candidate) {
+    const words = candidate.candidate.split(' ');
+
+    if (words.length < 7) {
+        console.error('Invalid ice candidate!', candidate);
+        return false;
+    }
+
+    // https://datatracker.ietf.org/doc/html/rfc5245#section-15.1
+    const transport = words[2];
```

Comment:
The lines of code before the switch case are common to both `shouldSend` and `shouldAccept` so we should create a new function for those and move it there